pkg/services/ngalert/notifier/config.go

```diff
@@ -78,7 +78,7 @@ func PersistTemplates(cfg *api.PostableUserConfig, path string) ([]string, bool,
 func Load(rawConfig string) (*api.PostableUserConfig, error) {
```

Comment:
Would be great if we could see an end-to-end test that would cover:

- Startup of the Alertmanager and its API
- Config is received and stored in the database, a moment later it is picked up by the alertmanager and applied
- Stored config and running config are the same